### PR TITLE
Refactor: Remove trailing whitespace from strings

### PR DESCRIPTION
**Pull request by eumiro**

### CODE REVIEW

File: airflow/providers/cncf/kubernetes/operators/pod.py [hunk 1]
```diff
@@ -819,7 +819,7 @@ def process_pod_deletion(self, pod: k8s.V1Pod, *, reraise=True):
                     self.log.info("Deleting pod: %s", pod.metadata.name)
                     self.pod_manager.delete_pod(pod)
                 elif should_delete_pod and istio_enabled:
-                    self.log.info("Deleting istio-proxy sidecar inside %s: ", pod.metadata.name)
+                    self.log.info("Deleting istio-proxy sidecar inside %s:", pod.metadata.name)
```
Comment (uranusjr) [inline airflow/providers/cncf/kubernetes/operators/pod.py]:
```suggestion
                    self.log.info("Deleting istio-proxy sidecar inside: %s", pod.metadata.name)
```

This seems to better match the format in other messages

File: airflow/providers/google/cloud/transfers/facebook_ads_to_gcs.py [hunk 2]
```diff
@@ -149,7 +149,7 @@ def execute(self, context: Context):
             )
             raise AirflowException(message)
         total_row_count = self._decide_and_flush(converted_rows_with_action=converted_rows_with_action)
-        self.log.info("Facebook Returned %s data points in total: ", total_row_count)
+        self.log.info("Facebook Returned %s data points in total:", total_row_count)
```
Comment (uranusjr) [inline airflow/providers/google/cloud/transfers/facebook_ads_to_gcs.py]:
```suggestion
        self.log.info("Facebook Returned %s data points in total.", total_row_count)
```

I don’t even understand why the colon is there…

File: airflow/secrets/local_filesystem.py [hunk 3]
```diff
@@ -238,7 +238,7 @@ def load_variables(file_path: str) -> dict[str, str]:
     if invalid_keys:
         raise AirflowException(f'The "{file_path}" file contains multiple values for keys: {invalid_keys}')
     variables = {key: values[0] if isinstance(values, list) else values for key, values in secrets.items()}
-    log.debug("Loaded %d variables: ", len(variables))
+    log.debug("Loaded %d variables:", len(variables))
```
Comment (uranusjr) [inline airflow/secrets/local_filesystem.py]:
```suggestion
    log.debug("Loaded %d variables.", len(variables))
```

**Review by uranusjr — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Review by hussein-awala — APPROVED** [review]

**Comment by eladkal** [comment]
Needs rebase and resolve conflicts

**Comment by eladkal** [comment]
@eumiro can you handle it? I'd like to merge this one

**Review by uranusjr — APPROVED** [review]

**Comment by eladkal** [comment]
lets see if https://github.com/apache/airflow/pull/34918 fix the build issue

**Comment by uranusjr** [comment]
Tests don’t agree with the change

**Pull request closed without merging**